layers/+spacemacs/spacemacs-evil/funcs.el

```diff
@@ -84,16 +84,20 @@ Otherwise, revert to the default behavior (i.e. enable `evil-insert-state')."
        (or (not (fboundp 'evil-mc-get-cursor-count))
            (eq (evil-mc-get-cursor-count) 1))))
 
-(defun spacemacs/evil-mc-paste-after (&optional count register)
-  "Disable paste transient state if there is more that 1 cursor."
-  (interactive "p")
-  (if (spacemacs//paste-transient-state-p)
-      (spacemacs/paste-transient-state/evil-paste-after)
-    (evil-paste-after count (or register evil-this-register))))
+(defun spacemacs/evil-mc-paste-after (fn &rest args)
+   "Disable paste transient state if there is more that 1 cursor."
+   (interactive "p")
+   (if (spacemacs//paste-transient-state-p)
+       (spacemacs/paste-transient-state/evil-paste-after)
+     (apply fn (first args) (or (second args) evil-this-register) (last args))))
 
-(defun spacemacs/evil-mc-paste-before (&optional count register)
+(advice-add 'evil-paste-after  :around  #'spacemacs/evil-mc-paste-after)
```

Comment:
There are two spaces before and after `:around`, is this just a style choice to make it more readable, or should there only be one? The same thing applies to the evil-paste-before advice below.
Searching through the .emacs.d folder found 5 other occurrences of `advice-add` ... `:around`, and they only have 1 space around them.